python/hail/api1/dataset.py

```diff
@@ -2363,7 +2364,27 @@ def grm(self):
         :rtype: :py:class:`KinshipMatrix`
         """
 
-        return KinshipMatrix(Env.hail().methods.GRM.apply(self._jvds))
+        print("Computing GRM...")
```

Comment:
I like logging here, but use `info` so it goes through java to the log file too.

Reply:
oh, this is API1. nevermind